ups/qserv.table

```diff
@@ -17,4 +17,5 @@ setupRequired(xrootd)
 
 envPrepend(LD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
 envPrepend(PATH, ${PRODUCT_DIR}/bin)
+envPrepend(PATH, ${MYSQLCLIENT_DIR}/bin)
```

Comment:
This does not look right, there is no dependency on `mysqlclient` package in this table file, so MYSQLCLIENT_DIR will not be defined?